data/projects.json

```
@@ -248,5 +248,15 @@
         "level": "basic",
         "site": "",
         "tags": "score-Keeper, HTML, CSS, JAVASCRIPT"
+    },   
+    {
+        "name": "FNDetectoApp",
+        "display": "FNDetectoApp",
```

You could use the `display` with another tex.t
```suggestion
        "display": "Fake news detection",
```